libs/openFrameworks/utils/ofFileUtils.cpp

```diff
@@ -28,19 +28,19 @@ ofBuffer::ofBuffer()
 }
 
 //--------------------------------------------------
-ofBuffer::ofBuffer(const char * _buffer, std::size_t size)
-:buffer(_buffer,_buffer+size)
+ofBuffer::ofBuffer(const char * _buffer, std::size_t _size)
```

Comment:
I think we might need to be careful with naming variables with leading underscores, as, according to the C++ Standard, 17.4.3.1.2 Global Names [lib.global.names], paragraph 1:

> Each name that begins with an underscore is reserved to the implementation for use as a name in the global namespace.

see also mozilla's guidelines for variable naming
https://developer.mozilla.org/en-US/docs/Mozilla/Using_CXX_in_Mozilla_code

I assume the leading underscores were inserted for consistency with the old code, but it's probably safer to remove the leading underscores alltogether.

Reply:
Removing would be fine as well. I was going for consistency as it make it easier to handle the buffer conversion with SWIG when the arguments are named the same.

Reply:
"_buffer" & "_size" now "buffer" & "size"